src/cpp/session/modules/SessionRCompletions.R

```diff
@@ -2870,7 +2870,7 @@ assign(x = ".rs.acCompletionTypes",
             {
                # otherwise infer from the object
                types <- vapply(completions, FUN.VALUE = numeric(1), USE.NAMES = FALSE, function(i) {
-                  .rs.getCompletionType(object[[i]])
+                  tryCatch(.rs.getCompletionType(object[[i]]), error = function(e) .rs.acCompletionTypes$UNKNOWN)
```

Comment:
My main concern here is that using `tryCatch()` in a loop could be expensive if we have a large number of completions. Maybe we cold let `.rs.getCompletionType()` return `UNKNOWN` for a `NULL` / other unexpected input?